.github/workflows/ci.yml

```diff
@@ -36,6 +36,53 @@ jobs:
         run: tox
 
 
+  pulp_integration:
+    runs-on: ubuntu-20.04
+    name: Pulp Integration - ${{ matrix.py_version.name }}
+
+    env:
+      TOXENV: ${{ matrix.py_version.tox_env }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        py_version:
+          - name: '3.10'
+            tox_env: pulp-integration-py310
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install Python ${{ matrix.py_version.name }}
+        uses: actions/setup-python@v2
+        with:
+          py-version: ${{ matrix.py_version.name }}
```

Comment:
This should be `python-version`- note the warning at https://github.com/Shrews/ansible-builder/runs/7134536766?check_suite_focus=true#step:3:1 . It's currently doing the right thing, but only because the default `python-version: 3.x` the action declares currently resolves to `3.10`.

Reply:
Oopsie. Done